matroid.gemspec

```diff
@@ -30,7 +30,18 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.add_dependency 'omniauth-oauth2', '~> 1.3.1'
+  spec.add_dependency 'rest-client', '~> 1.8'
+
   spec.add_development_dependency "bundler", "~> 1.14"
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_runtime_dependency 'httparty', "~> 0.14.0"
+  spec.add_development_dependency "minitest"
+  spec.add_development_dependency "vcr"
+  spec.add_development_dependency "fakeweb"
+  spec.add_development_dependency "rspec"
+
+  spec.add_dependency "dotenv"
```

Comment:
Yes, but if they don't know that we will parse their `.env` file into local environment variables, they could run into some subtle bugs. Some people use `.env` for production deployment, but the production variables might not be appropriate in a situation where they're using our API locally. 

It's better not to surprise the users; after we discuss environment variables in `README.md` we should explicitly mention the `dotenv` gem.